docs/reference/configuration/external dependencies/irsa-configuration.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Configure UDS Core to Use IAM Roles for Service Accounts (IRSA) 
+---
+
+Several applications within UDS Core can be configured to utilize resources that are external to your Kubernetes cluster, such as object storage and databases. If you are running in AWS, you can leverage [IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to provide applications within UDS Core with a secure means of accessing external resources.
+
+The following applications in UDS Core that support IRSA are:
+- [Loki](#loki)
+- [Velero](#velero)
+
+:::note
+There may be additional applications in UDS Core that are not mentioned in this list that support IRSA. Only the applications listed above have been validated to work within UDS Core.
+:::
+
+This guide will cover how to configure IRSA for each application. 
+
+## Prerequisites
+Configuring IRSA requires that you have configured an IAM OIDC provider for your cluster. Refer to the IRSA [documentation](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for more information.
+
+You must also create IAM Roles and Policies for each application. Refer to the IRSA [documentation](https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html) for more information.
+
+## Bundle Configuration
+Configuring applications within UDS Core to use IRSA requires that you declare overrides in UDS Bundle configuration. Below are the necessary overrides for each application.
+
+:::note
+The examples below are not exhaustive representations of all of the values you will need to supply to configure external storage.
+:::
+
+:::note
+The examples below declare new `variables` for IRSA Role ARN annotations, allowing for these values to be passed in dynamically via `uds-config.yaml` as opposed to being hardcoded as `overrides`.
+:::
```

Comment:
nit: the back to back note blocks feels weird, does it make sense to combine these?